src/Geno/C++/Components/BuildMatrix.cpp

```diff
@@ -32,13 +33,13 @@ void BuildMatrix::NewColumn( std::string Name )
 
 //////////////////////////////////////////////////////////////////////////
 
-void BuildMatrix::NewConfiguration( std::string_view WhichColumn, std::string Configuration )
+void BuildMatrix::NewConfiguration( std::string_view WhichColumn, std::string configuration )
```

Comment:
Was this an actual problem on Linux? Where could it not figure out to use the variable instead of the class?